modules/ROOT/pages/websockets/websocket-connector-server-components.adoc

```diff
@@ -0,0 +1,55 @@
+= About Websocket Server Components
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket
+
+== About Websocket Server Components
+
+=== Inbound Listener
+
+The Inbound Listener component represents an websocket endpoint that clients can connect to. By hitting this endpoint, external clients not only establish a Websocket connection, they will trigger the owning server flow each time they send a message.
+
+[source,xml,linenums]
+----
+<flow name="onIncomingMessage">
+    <websocket:inbound-listener ​path​=​"/quotes"​ ​defaultGroups​=​"#[['AVENGERS', 'REVENGERS']]"​ ​config-ref​=​"ws"​ />
+    <set-variable name="myMessage" value="#[payload]" />
```

Comment:
we advise customers not to put so many set-variable in a row. If you want to do this example, then use ee:transform.

That been said, this is not a very nice way of communicating this. You should describe the attributes object here and explain the use cases for each of its properties.